docs/Github-workflow.md

```diff
@@ -0,0 +1,61 @@
+[Back to overview](https://github.com/WordPress/performance/blob/trunk/docs/README.md)
+
+# GitHub workflow for the Performance Lab plugin
+
+The [Performance team](https://make.wordpress.org/core/2021/10/12/proposal-for-a-performance-team/) uses GitHub to manage all code and related discussions for the Performance Lab plugin. Please follow the workflow below to ensure that all issues are properly tracked.
+
+## Issues
+When [opening a new issue](https://github.com/WordPress/performance/issues/new/choose), use the appropriate template: Bug report, Feature request, or Report a security vulnerability. All new issues should include the following labels:
+
+- A _[Focus]_ label, or Infrastructure if the issue relates to the plugin infrastructure. The [Focus] labels are aligned with the Performance team's [focus areas](https://docs.google.com/spreadsheets/d/16N5oZ9wE6AkiqMz7b_707eh24vvpjMwsEG67XFAbxy8/edit#gid=0) and [GitHub Projects](https://github.com/WordPress/performance/projects).
+- A _[Type]_ label
+- A _[Module]_ label if the issue relates to an existing module
+
+In addition, the new issue should be assigned to an appropriate Project. By default, new issues will automatically be added to the Backlog column within its project. This is intended for tracking any future work that is not currently a high priority as defined by the project’s point(s) of contact (POCs). Contributors are welcome to work on issues in the Backlog, but response rates may be slower than they are on prioritized issues.
+
+## Working on an issue
```

Comment:
I think this belongs under the "Issues" heading or not? In other words, I'd make this a `h3`, and then you could make all the labels below `h4`.